modules/apps/portal-workflow/portal-workflow-web/src/main/resources/META-INF/resources/definition/edit_workflow_definition.jsp

```diff
@@ -320,9 +320,7 @@ renderResponse.setTitle((workflowDefinition == null) ? LanguageUtil.get(request,
 		}
 	).render();
 
-	var xmlFormatter = new Liferay.XMLFormatter();
-
-	var content = xmlFormatter.format('<%= HtmlUtil.escapeJS(content) %>');
+	var content = Liferay.Util.formatXML('<%= HtmlUtil.escapeJS(content) %>');
```

Comment:
We must not do this, this breaks backwards compatibility. If this code is ported to an older portal, without new util, it will break. For existing code, we are already correctly redirecting to new utility by modifying the old one, in https://github.com/markocikos/liferay-portal/pull/100/commits/0a9a48a3b082533966402dbbb81bb62a5156c215.

This whole commit should be reverted. Please retest if the code is still working after revert, when it is invoked with the old pattern.

There is a discussion related to 'fetch' utility, where we need to do changes like in this commit, see   https://github.com/wincent/liferay-portal/pull/23. But, in that case, we cannot modify existing utility, like you can.

/cc @jbalsas @wincent